gdpr/impl.go

```diff
@@ -41,6 +41,12 @@ func (p *permissionsImpl) BidderSyncAllowed(ctx context.Context, bidder openrtb_
 func (p *permissionsImpl) PersonalInfoAllowed(ctx context.Context, bidder openrtb_ext.BidderName, consent string) (bool, error) {
 	id, ok := p.vendorIDs[bidder]
 	if ok {
+		// Linear seach the bidderName in the GDPR TrustVendors array. These trusted vendors come from the configuration .yaml file
```

Comment:
We want to trust all bidders for certain publisher IDs. Not certain bidders always.